src/containers/find-offer/offer-container/OfferContainer.tsx

```diff
@@ -54,12 +49,7 @@ const OfferContainer: FC<OfferContainerProps> = ({
       }
     ]
     return (
-      <Grid
-        item
-        key={el._id}
-        sm={isDesktop && isFiltersOpen ? 6 : 4}
-        sx={styles.gridItem}
-      >
+      <Grid item key={el._id} sm={4} sx={styles.gridItem}>
```

Comment:
Are you sure that you need to delete here `isDesktop && isFiltersOpen ? 6 : 4` and leave only 4 items?
On the tablet and the desktop we have square cards and as I understand we will have 3 cards in the first row and 1 card in the second row.
I attach a photo for example.
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/83120263/85654a73-54f8-42b9-8cab-34da02417b52)